csrankings-s.csv

```diff
@@ -1127,6 +1127,7 @@ Shuqun Zhang,CUNY,http://www.cs.csi.cuny.edu/~zhangs,Uhd766gAAAAJ
 Shuran Song,Columbia University,http://shurans.github.io,5031vK4AAAAJ
 Shurui Zhou,University of Toronto,https://www.eecg.utoronto.ca/~shuruiz,6ej1Rm4AAAAJ
 Shuvra S. Bhattacharyya,University of Maryland - College Park,https://www.umiacs.umd.edu/people/ssb,rNpUIKAAAAAJ
+Shuyin Jiao,North Carolina State University,https://www.csc.ncsu.edu/people/sjiao2,NOSCHOLARPAGE

```

Comment:
Teaching faculty not eligible for inclusion.

Reply:
Sorry about that, fixed with [1ec484a](https://github.com/emeryberger/CSrankings/pull/5450/commits/1ec484acd42d7409b02f16a1f84a3fc4baf4ef71) bellow in the pull request.